### PR TITLE
W-17281517: Upgrade jakarta dependencies

### PR DESCRIPTION
**Pull request by marianorean**

### CODE REVIEW

File: modules/extensions-spring-support/pom.xml [hunk 1]
```diff
@@ -432,5 +432,10 @@
             <classifier>mule-plugin</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
```
Comment (elrodro83) [inline modules/extensions-spring-support/pom.xml:437]: this is already included: https://github.com/mulesoft/mule/blob/master/modules/extensions-spring-support/pom.xml#L153

File: modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/capability/xml/schema/model/package-info.java [hunk 2]
```diff
@@ -11,6 +11,6 @@
 // Generated on: 2011.06.14 at 03:58:12 PM GMT-03:00 
 //
 
-@javax.xml.bind.annotation.XmlSchema(namespace = "http://www.w3.org/2001/XMLSchema",
-    elementFormDefault = javax.xml.bind.annotation.XmlNsForm.QUALIFIED)
+@jakarta.xml.bind.annotation.XmlSchema(namespace = "http://www.w3.org/2001/XMLSchema",
+    elementFormDefault = jakarta.xml.bind.annotation.XmlNsForm.QUALIFIED)
```
Comment (elrodro83) [inline modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/capability/xml/schema/model/package-info.java:15]: static import

File: tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/AbstractOSController.java [hunk 3]
```diff
@@ -53,7 +53,7 @@ public abstract class AbstractOSController {
    * These values represent the status returned by running `mule status` for the wrapper and Java processes respectively.
    */
   public enum MuleProcessStatus {
-    STARTING_STARTING, STARTED_STARTING, STARTED_STARTED, STARTING_LAUNCH, STARTING_LAUNCHING
+    STARTING_STARTING, STARTED_STARTING, STARTED_STARTED, STARTING_LAUNCH, STARTING_LAUNCHING, STARTING_STOPPING, STARTING_STOPPED, STOPPING_DOWN
```
Comment (elrodro83) [inline tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/AbstractOSController.java:56]: is this related to this change?

File: tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/Controller.java [hunk 4]
```diff
@@ -103,12 +103,18 @@ public void printLog() throws IOException {
       final StringBuilder muleLogBuilder = new StringBuilder();
       muleLogBuilder.append(muleLogFile.getName() + ":" + lineSeparator());
       muleLogBuilder.append("============" + lineSeparator());
+      if (appsDir.toString().contains("AppControlTestCase")) {
```
Comment (elrodro83) [inline tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/Controller.java:106]: this dones't seem right

File: tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/AbstractOSController.java (hunk 3 above)
Comment (marianorean) [reply tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/AbstractOSController.java:56]: was added for debugging a failling test, I'll remove it

File: tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/Controller.java (hunk 4 above)
Comment (marianorean) [reply tests/infrastructure/src/main/java/org/mule/test/infrastructure/process/Controller.java:106]: Same as before

File: modules/extensions-spring-support/pom.xml (hunk 1 above)
Comment (marianorean) [reply modules/extensions-spring-support/pom.xml:437]: it is necessary to add it with test scope, otherwise PetStoreCachedConnectionWithDynamicParameterTestCase fails

**Review by elrodro83 — APPROVED** [review]
